azure/template.json

```diff
@@ -120,6 +120,9 @@
     "deploySupportApp": {
       "type": "bool"
     },
+    "oltdNotificationToEmployerJobSchedule": {
```

Comment:
can we remove this and `oltdNotificationToServiceDeskJobSchedule` and handle these in table storage config as previously discussed.